spec/policies/folder_policy_spec.rb

```diff
@@ -32,6 +33,14 @@
     it 'can destroy folder' do
       assert_permit bob, folder2, :destroy?
     end
+
+    it 'cannot destroy folder when in personal-team' do
```

Comment:
should be possible to destroy all the folders.
if inbox does not exist, we are creating it